storage/migrations/17_consensus_tx_index.up.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
```

Comment:
Added these indexes might take non-trivial time. Do we have any special requirements or handling of such migrations that aren't super quick? 

The specific changes in this migration are not breaking and could actually be made whenever (don't need to happen at deployment), so these can be handled that way, but i'm asking more for any general guidelines.

Reply:
Thanks for the info. For now I updated just update the migration to do `CREATE INDEX IF NOT EXISTS` so that we are more flexible on how to handle this deployment.